tests/e2e/auth/vesting/suite.go

```diff
@@ -221,6 +223,11 @@ func (s *E2ETestSuite) TestNewMsgCreatePermanentLockedAccountCmd() {
 }
 
 func (s *E2ETestSuite) TestNewMsgCreateClawbackVestingAccountCmd() {
+	pwd, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println(pwd)
```

Comment:
```suggestion
```

Reply:
oh we need this, just the print can go away